### ISSUE TITLE
Script does not reset binning

### ISSUE THREAD

**Issue by joshwalawender** [KeckObservatory/LRIS_Focus_GUI#5]
Observers tonight reported that the script did not reset the binning values after completion.

**Comment by lucarizzi** [comment]
This is true but it was always true. The old focus script reset the binning to 1x1 and left it there so I modeled on that but it’s easy to change.

**Comment by joshwalawender** [comment]
Yeah, I figured as much.  Observers tonight ended up taking calibrations in the wrong binning.  I think it makes sense to restore binning.
